r/README.md

```diff
@@ -143,7 +143,15 @@ checkout:
 
 ``` shell
 cd ../../r
-R -e 'install.packages(c("devtools", "roxygen2", "pkgdown", "covr")); devtools::install_dev_deps()'
+
+R --vanilla << EOF
+options(repos = c(REPO_NAME = "https://cloud.r-project.org/"))
+install.packages(c("bit64", "devtools", "roxygen2", "tidyselect",
```

Comment:
bit64 and tidyselect are hard dependencies of arrow so you don't need to install them explicitly

None of the other dependencies are required to build the package, and they bring a lot of stuff with them, so it's probably better to leave them off as a separate step.

The quickest version of this is `install.packages("remotes"); remotes::install_deps(dependencies = TRUE)`